api/src/main/java/org/jboss/forge/roaster/model/source/AnnotationTargetSource.java

```diff
@@ -51,4 +51,9 @@
     * Remove an annotation instance from this {@link T}.
     */
    T removeAnnotation(Annotation<O> annotation);
+   
+   /**
+    * Remove all annotations instance from this {@link T}.
+    */
+   void removeAnnotations();
```

Comment:
Why not... I prefer annotationS because of auto-complete in IDE. If you tipe removeAnno... you'll see removeAnnotation() & removeAnnotations but not removeAllAnnotation.
These are just details ^_^